plugins/catalog-react/src/index.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from '@backstage/catalog-client';
```

Comment:
Previously it was exported that way by `@backstage/plugin-catalog` so I just kept it to be compatible.

So in the future the idea would be to rename it to `@backstage/plugin-catalog-common`, right? We said it would be handy if `@backstage/plugin-catalog-react` would reexport that package, or should we take care to only export parts of it?

Reply:
Yeah I'm torn. If common contains an actual implementation of something that users are only meant to consume via API/interface - it doesn't make sense to export it to people that read it. Just like we wouldn't export it publicly if it was in the plugin itself

Reply:
Maybe we should remove the [export here](https://github.com/backstage/backstage/blob/ad364bdf575a891ee43a0b49ff8cc1046f0f0bee/packages/catalog-client/src/index.ts#L17), have a second file that exports the implementation for the cases we need it (I hope this works, or can I just import the `index.ts`?) and keep the export here as-is.

Reply:
I don't think that's an option right? That's the exact one that people want to be able to `import { CatalogClient } from '@backstage/catalog-client'`

Reply:
What I do NOT want people to do, is `import { CatalogClient } from '@backstage/catalog-common-react'`

Reply:
To be more clear - if we change your line to just `export { CatalogApi } from '@backstage/catalog-client'`, I think we'd be set. Does that makes sense?

Reply:
The more I think about it, the less I think that catalog-client should be renamed to `plugin-catalog-common` as the package is quite special. If we don't plan to do that, I think limiting the export would be the way to go. Already pushed that change 😉

Reply:
Cool! Now an import broke tho. I think the rest is good to go, give me a sec

Reply:
Arg, ... I guess that test would be better without the CatalogClient, but that is not something I want to fix now...